src/main-menu.ts

```diff
@@ -0,0 +1,44 @@
+import { LitElement, css, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { game } from './game';
+
+@customElement('main-menu')
+export class MainMenu extends LitElement {
+  static styles = css`
+    :host {
+      width: 100vw;
+      height: 100vh;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      flex-direction: column;
+    }
+  `;
+
+  @property({ type: Boolean })
+  settingsOpen = false;
+
+  startGame() {
+    this.remove();
+    game.start();
+  }
+
+  renderSettings() {
+    return html`
+      <h1> Settings </h1>
+      <button @click=${() => (this.settingsOpen = false)}> Back </button>
+    `;
+  }
+
+  render() {
+    if (this.settingsOpen) {
+      return this.renderSettings();
+    }
+
+    return html`
+      <h1> Power Lifter </h1>
+      <button @click=${this.startGame}> Play </button>
```

Comment:
Does this need arrow notation?

Reply:
Nope! 

https://lit.dev/docs/components/events/#adding-event-listeners-in-the-element-template